lib/auth/auth_test.go

```diff
@@ -2701,6 +2702,25 @@ func TestCAGeneration(t *testing.T) {
 	}
 }
 
+func TestGetLicense(t *testing.T) {
+	suite := newAuthSuite(t)
```

Comment:
I would rename this `s` or something to not conflict with imports:
`Variable 'suite' collides with imported package name `